odl/test/largescale/tomo/ray_transform_slow_test.py

```diff
@@ -11,52 +11,61 @@
 from __future__ import division
 
 import numpy as np
-from packaging.version import parse as parse_version
 import pytest
+from packaging.version import parse as parse_version
 
 import odl
-from odl.util.testutils import skip_if_no_largescale, simple_fixture
-from odl.tomo.util.testutils import (skip_if_no_astra, skip_if_no_astra_cuda,
-                                     skip_if_no_skimage)
+from odl.tomo.util.testutils import (
+    skip_if_no_astra, skip_if_no_astra_cuda, skip_if_no_skimage)
+from odl.util.testutils import simple_fixture, skip_if_no_largescale
 
 
 # --- pytest fixtures --- #
 
 
+pytestmark = skip_if_no_largescale
+
+
 dtype_params = ['float32', 'float64', 'complex64']
 dtype = simple_fixture('dtype', dtype_params)
 
 
 # Find the valid projectors
-projectors = [skip_if_no_astra('par2d astra_cpu uniform'),
-              skip_if_no_astra('par2d astra_cpu nonuniform'),
-              skip_if_no_astra('par2d astra_cpu random'),
-              skip_if_no_astra('cone2d astra_cpu uniform'),
-              skip_if_no_astra('cone2d astra_cpu nonuniform'),
-              skip_if_no_astra('cone2d astra_cpu random'),
-              skip_if_no_astra_cuda('par2d astra_cuda uniform'),
-              skip_if_no_astra_cuda('par2d astra_cuda nonuniform'),
-              skip_if_no_astra_cuda('par2d astra_cuda random'),
-              skip_if_no_astra_cuda('cone2d astra_cuda uniform'),
-              skip_if_no_astra_cuda('cone2d astra_cuda nonuniform'),
-              skip_if_no_astra_cuda('cone2d astra_cuda random'),
-              skip_if_no_astra_cuda('par3d astra_cuda uniform'),
-              skip_if_no_astra_cuda('par3d astra_cuda nonuniform'),
-              skip_if_no_astra_cuda('par3d astra_cuda random'),
-              skip_if_no_astra_cuda('cone3d astra_cuda uniform'),
-              skip_if_no_astra_cuda('cone3d astra_cuda nonuniform'),
-              skip_if_no_astra_cuda('cone3d astra_cuda random'),
-              skip_if_no_astra_cuda('helical astra_cuda uniform'),
-              skip_if_no_skimage('par2d skimage uniform')]
-
-projector_ids = [" geom='{}' - impl='{}' - angles='{}' "
-                 ''.format(*p.args[1].split()) for p in projectors]
-
-
-# bug in pytest (ignores pytestmark) forces us to do this this
-largescale = " or not pytest.config.getoption('--largescale')"
-projectors = [pytest.mark.skipif(p.args[0] + largescale, p.args[1])
-              for p in projectors]
+projectors = []
+projectors.extend(
+    (pytest.param(value, marks=skip_if_no_astra)
+     for value in ['par2d astra_cpu uniform',
```

Comment:
Prefer something other than "value", e.g. be specific.

Reply:
Well, `values` is the name under which the input is stored in `pytest.param`. But it's a generic construct, so you're right, we can be somewhat more specific here.